src/includes/debug-symbols-apple/_default.mdx

```diff
@@ -167,7 +167,13 @@ You need to have an Auth Token for this to work. You can [create an Auth Token h
 1.  Download and install [sentry-cli](/product/cli/installation/)
 2.  You will need to copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_
 
-```bash
+<Alert level="" title="Warnings vs. errors">
```

Comment:
Making this title case. Also added an empty line between the Alert tag and the markdown text. This ensures any formatting in your markdown renders properly.

```suggestion
<Alert level="" title="Warnings vs. Errors">

```